internal/host/common.go

```diff
@@ -19,17 +19,18 @@ import (
 )
 
 const (
-	statusInfoDisconnected               = "Host keepalive timeout"
-	statusInfoDisabled                   = "Host is disabled"
-	statusInfoDiscovering                = "Waiting for host hardware info"
-	statusInfoInsufficientHardware       = "Host does not pass minimum hardware requirements"
-	statusInfoPendingForInput            = "User input required"
-	statusInfoNotReadyForInstall         = "Host not ready for install"
-	statusInfoInstalling                 = "Installation in progress"
-	statusInfoResettingPendingUserAction = "Reboot the host into the installation image to complete resetting the installation"
-	statusInfoPreparingForInstallation   = "Preparing host for installation"
-	statusInfoPreparingTimedOut          = "Cluster is no longer preparing for installation"
-	statusInfoAbortingDueClusterErrors   = "Installation has been aborted due cluster errors"
+	statusInfoDisconnected               = "Host has stopped communicating with the installation service"
+	statusInfoDisabled                   = "Host was disabled by user"
```

Comment:
```suggestion
	statusInfoDisabled                   = "Host was manually disabled"
```

At least in the UI we generally try to avoid calling users "users" 🙂.

Reply:
Done